code/datums/traits/neutral.dm

```diff
@@ -97,3 +97,21 @@
 
 /datum/quirk/monochromatic/remove()
 	quirk_holder.remove_client_colour(/datum/client_colour/monochrome)
+
+//MonkeStation Edit Start - Bald Quirk
+/datum/quirk/bald
```

Comment:
consider moving this to its own modular file

Reply:
Didn't even catch that. Yeah, move it to a module file, quick and easy.